docs/index.md

```diff
@@ -1653,6 +1659,13 @@ describe('my API', function() {
 
 Finally, use this command to bring it together: `mocha --require fixtures.mjs test.spec.mjs`.
 
+## Test Fixture Decision-Tree Wizard Thing
+
+This flowchart will help you decide which of [hooks], [root hook plugins] or
+[global fixtures] you should use.
+
+{% include fixture-wizard.html %}
```

Comment:
Including HTML is not handled properly and [SVG is broken](https://deploy-preview-4440--mocha.netlify.app/#test-fixture-decision-tree-wizard-thing). 
I don't know why even I checked it.

Reply:
ohh I know what happened.  prettier broke it.  I am not sure if this is a bug in prettier or liquid or eleventy, but when prettier reformats it, the svg breaks.

Reply:
OK, wrapping the svg in a div works.  I think this is an 11ty bug

Reply:
see https://github.com/11ty/eleventy/issues/1442